cmd/edit.go

```diff
@@ -46,7 +46,7 @@ func newEditProjectCmd() *cobra.Command {
 		Run: func(cmd *cobra.Command, args []string) {
 			dieIfNoProject()
 
-			projectInfo, err := scaffold.LoadProjectFile("PROJECT")
+			projectInfo, err := scaffoldutil.LoadProjectFile("PROJECT")
```

Comment:
IHMO: we could keep the alias as utils only.
```suggestion
			projectInfo, err := utils.LoadProjectFile("PROJECT")
```
Has any other utils in the project?

Reply:
### About the alias
There are at least two `util` packages:
1. `cmd/util`
2. `pkg/scaffold/util`

That's the reason why I used `scaffoldutil` as the alias in the files in the `cmd` package: `cmd/edit.go`, `cmd/main.go`, `cmd/webhook_v1.go` and `cmd/webhook_v2.go`.

### About the package name
I used `util` because it already existed. It contains the following functions:
1. `GetResourceInfo`: the `v2` version of this function (there is a different one for `v1` as paths are different. #1255 moves this function to the `resource` package.
2. `LoadProjectFile`: introduced in this PR as a temporary place. I will make a PR moving this function and all the project configuration related things to `internal/config`.

When both functions are moved, the `pkg/scaffold/util` package will be empty, so the name will not matter.

Reply:
Update: #1325 moves the remaining function out of `pkg/scaffold/util`, so this should not be a stopper for this PR.